.github/workflows/gateway.yml

```diff
@@ -31,7 +31,10 @@ jobs:
       - uses: ./.github/actions/untracked
       - uses: ./.github/actions/setup-python
       - name: Install dependencies
+        # The pydantic pin below is temporary until FastAPI releases a compatible build for the
```

Comment:
Is FastAPI related?

Reply:
That's where the actual Exception is raised from (including a type that is no longer supported in the `fields` module within pydantic)

Reply:
> That's where the actual Exception is raised from

link?

Reply:
Is there an issue in the fastapi repo related to this?